### ISSUE TITLE
[Color contrast tool] TextBoxes should be wider

### ISSUE THREAD

**Issue by niels9001** [microsoft/WinUI-Gallery#1265]
### Which version of the app?

WinUI 3 Gallery

### Description

When the color code textboxes have focus, the clear button appears and the text is cut off.. which is annoying as changing the cursor position will make the text jump around.

Increasing the width to support a hex code + clear button should resolve that.

<img width="358" alt="image" src="https://user-images.githubusercontent.com/9866362/231422332-0342807b-c256-4ab9-9ee6-30ac132cc71a.png">
